recipe/meta.yaml

```diff
@@ -1,24 +1,28 @@
-{% set version = "0.5.4" %}
+{% set name = "isodate" %}
+{% set version = "0.6.0" %}
+{% set sha256 = "2e364a3d5759479cdb2d37cce6b9376ea504db2ff90252a2e5b7cc89cc9ff2d8" %}
```

Comment:
@nehaljwani I really dislike that excessive use of Jinja. Can you revert those changes and keep the recipe simple and readable?

Reply:
:thinking: Removal requires extra effort after `conda skeleton` :sob:

Reply:
Argh, those were added in the skeleton? That is really bad taste IMO.
I always prefer readability vs apparent practicality.

Reply:
Just tried that with conda skeleton and I don't see these extra jinjing:

```yaml
package:
  name: isodate
  version: "0.6.0"

source:
  fn: isodate-0.6.0.tar.gz
  url: https://pypi.python.org/packages/b1/80/fb8c13a4cd38eb5021dc3741a9e588e4d1de88d895c1910c6fc8a08b7a70/isodate-0.6.0.tar.gz
  md5: 0e1203fce27ce65e2d01c5f21c4d428f
#  patches:
   # List any patch files here
   # - fix.patch

# build:
  # noarch: python
  # preserve_egg_dir: True
  # entry_points:
    # Put any entry points (scripts to be generated automatically) here. The
    # syntax is module:function.  For example
    #
    # - isodate = isodate:main
    #
    # Would create an entry point called isodate that calls isodate.main()


  # If this is a new build for the same version, increment the build
  # number. If you do not include this key, it defaults to 0.
  # number: 1

requirements:
  build:
    - python
    - setuptools
    - six

  run:
    - python
    - six

test:
  # Python imports
  imports:
    - isodate
    - isodate.tests

  # commands:
    # You can put test commands to be run here.  Use this to test that the
    # entry points work.


  # You can also put a file called run_test.py in the recipe that will be run
  # at test time.

  # requires:
    # Put any additional test requirements here.  For example
    # - nose

about:
  home: https://github.com/gweis/isodate/
  license: BSD License
  summary: 'An ISO 8601 date/time/duration parser and formatter'
  license_family: BSD

# See
# http://docs.continuum.io/conda/build.html for
# more information about meta.yaml
```

Reply:
Try the latest conda build

Reply:
conda-forge is still conda-build <3 :unamused: